interp/type.go

```diff
@@ -483,7 +483,11 @@ func nodeType2(interp *Interpreter, sc *scope, n *node, seen []*node) (t *itype,
 			length = int(vInt(sym.rval))
 		default:
 			// Size is defined by a numeric constant expression.
-			if _, err = interp.cfg(c0, sc, sc.pkgID, sc.pkgName); err != nil {
+			if _, err := interp.cfg(c0, sc, sc.pkgID, sc.pkgName); err != nil {
+				if strings.Contains(err.Error(), " undefined: ") {
```

Comment:
this is horrible, but ok :)
Do you think we could a little bit more precise, and try to make sure that the error ends with the name of the undefined symbol? ("num" in the example).
I'm not sure it's possible though, because I've tried printing c0.ident and it seems it was empty at this point.
I'll let you be the judge and approve the PR anyway.

Reply:
We need an additional helper to format the offending name from the AST. In the simplest case it could be the node ident, but not always and I not fully clear about the complexity. I think it should be the subject of another PR, and that we can merge this one.